### PR TITLE
Fix for CI badge incorrect status

### LINKED ISSUE #124: CI Badge reports "no status" on master's readme
CI hasn't run on master branch so the badge is showing "no status":
Screenshot 2020-01-15 at 10 39 52

### PR DESCRIPTION
**Pull request by pozil**
Fixes #124 by forcing a CI run on master after a push

### CODE REVIEW

File: .github/workflows/scratch-org-sfdx-ci.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ name: Salesforce DX CI (scratch org only)
 
 # Definition when the workflow should run
 on:
+    push:
```
Comment (muenzpraeger) [inline .github/workflows/scratch-org-sfdx-ci.yml]: Not sure I understand this addition. `master` is push-protected.

**Comment by pozil** [comment]
Yes but this 'push' event is fired when PR are merged so that sets the
state of master.

**Comment by muenzpraeger** [comment]
I'm not aware that a merged `pull_request` kicks of a new `push`, @pozil. If you validated that it 's good to know.

Having said that, a new PR sets also the status (when `master` is the target branch). So even after a new PR gets merged, kicks off a good run, a following (failing) PR will re-set the badge. This solution won't work then. You can see that live as the GitHub action shows failing for [this PR](https://github.com/trailheadapps/ebikes-lwc/commit/3c855a438d8a7b7ac33471267e82f5b2695ededa/checks?check_suite_id=410300078).

**Comment by pozil** [comment]
Damn badges :(
Closing that PR for now since it doesn't completely fix the issue.

**Pull request closed without merging**